internal/hud/webview/convert.go

```diff
@@ -34,18 +34,18 @@ func StateToProtoView(s store.EngineState, logCheckpoint logstore.Checkpoint) (*
 		ms := mt.State
 
 		var absWatchDirs []string
-		var absWatchPaths []string
-		for _, p := range mt.Manifest.LocalPaths() {
+		for i, p := range mt.Manifest.LocalPaths() {
+			if i > 50 {
```

Comment:
fwiw, i think should replace this with some notion of a base dir for custom builds -- it doesn't make much sense for custom builds with 4000 file inputs